apps/api/package.json

```diff
@@ -72,6 +72,7 @@
     "pino": "^8.1.0",
     "pino-http": "^8.1.0",
     "pino-pretty": "^8.1.0",
+    "pins-data-model": "github:Planning-Inspectorate/data-model#draft-01",
```

Comment:
I'd expect a package-lock change along with this, also draft-02 is latest now

Reply:
Updated to draft-02 and the package-lock change is there now 👍